File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/emergency/shared/emergency-discharge-summary.model.ts
import {
    NgForm,
    FormGroup,
    FormControl,
    Validators,
    FormBuilder
} from '@angular/forms';

export class EmergencyDischargeSummary {
    public ERDischargeSummaryId: number = 0;
    public PatientId: number = null;
    public PatientVisitId: number = null;
    public DischargeType: string = null;
    public ChiefComplaints: string = null;
    public TreatmentInER: string = null;
    public Investigations: string = null;
    public AdviceOnDischarge: string = null;
    public OnExamination: string = null;
    public ProvisionalDiagnosis: string = null;
    public DoctorName: string = null;
    public MedicalOfficer: string = null;
    public CreatedOn: string = null;
    public CreatedBy: number = null;
    public ModifiedOn: string = null;
    public ModifiedBy: number = null;

    public DoctorSelected: any = null;
    public MedicalOfficerSelected: any = null;
}

File: Code/Components/DanpheEMR.ServerModel/EmergencyModels/EmergencyDischargeSummaryModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel.EmergencyModels
{
    public class EmergencyDischargeSummaryModel
    {
        [Key]
        public int ERDischargeSummaryId { get; set; }
        public int PatientId { get; set; }
        public int PatientVisitId { get; set; }
        public string DischargeType { get; set; }
        public string ChiefComplaints { get; set; }
        public string TreatmentInER { get; set; }
        public string Investigations { get; set; }
        public string AdviceOnDischarge { get; set; }
        public string OnExamination { get; set; }
        public string ProvisionalDiagnosis { get; set; }
        public string DoctorName { get; set; }
        public string MedicalOfficer { get; set; }
        public int? CreatedBy { get; set; }
        public DateTime? CreatedOn { get; set; }
        public int? ModifiedBy { get; set; }
        public DateTime? ModifiedOn { get; set; }
        
    }
}
